### ISSUE TITLE
encode a full LTC frame in reverse

### ISSUE THREAD

**Issue by superbigio** [x42/libltc#42]
Hi Robin,
I would also like to include a convenience function for that.

`void ltc_encoder_encode_frame_reversed(LTCEncoder *e);`

Trivial to add and I think it would make for a more symmetric API.
If you agree I can send you a PR. Thanks.

**Comment by x42** [comment]
So basically add

```
void ltc_encoder_encode_reversed_frame (LTCEncoder *e) {
  int byte;
  for (byte = 9 ; byte > 0 ; --byte) {                                                                                                                                                        
    encode_byte(e, byte, 1.0);
  }
}
```
sounds reasonable. I can add that, no PR needed

**Comment by superbigio** [comment]
Hi Robin,

thanks for adding the function.
However there is a little oversight in the for loop boundaries.
It should be:
for (byte = 9 ; byte >= 0 ; --byte)

- Luigi

**Comment by x42** [comment]
Oh my. Thanks for catching that. Fixed in ca108a665
